enable-scripts.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/.eslintignore b/.eslintignore
```

Comment:
This file is just a patch that will enable those scripts when applied.
I don't want to do it in this PR, because I would have to run `yarn run prettier-all` to make tests pass, but I think it should be done in another commit made especially for reformatting.